hpat/tests/test_series.py

```diff
@@ -186,7 +186,7 @@ def test_impl(A):
         S = pd.Series([3, 5, 6], ['a', 'b', 'c'], name='A')
         pd.testing.assert_series_equal(hpat_func(S), test_impl(S))
 
-    def test_series_attr1(self):
+    def test_series_size(self):
```

Comment:
We don't need to create DataFrame for this.
Please test at least two dataset: numeric (inlucde NaN) and strings

Reply:
Done